Packs/XSOARLabUpdates/README.md

```diff
@@ -0,0 +1,19 @@
+### This pack sends content-updates to slack channels.
```

Comment:
Can slightly twich the format the look like this? https://xsoar.pan.dev/docs/documentation/pack-docs#pack-readme